src/Features/Core/Portable/CodeLens/CodeLensReferencesService.cs

```diff
@@ -180,7 +179,13 @@ private static SyntaxNode GetEnclosingCodeElementNode(Document document, SyntaxT
 
                     await Task.WhenAll(referenceTasks).ConfigureAwait(false);
```

Comment:
Task.WhenAll only ensures that the tasks have completed, not that they completed successfully.  If cancellation happens the tasks will be in the Canceled state.  Calling .Result on a Canceled task will result in an AggregateException being thrown.